🌓 Dark mode fixes

michalsmiarowski opened the pull request:
Closes: #435 

Also fixes the dark mode issues that were placed in Coda (see [here](https://coda.io/d/Building-Keep_d-fmEgBNFVH/Sorin-2023-18-07-QA-tBTC-v2-Unmint-flow_suH93#_luYJg), [here](https://coda.io/d/Building-Keep_d-fmEgBNFVH/Sorin-2023-18-07-QA-tBTC-v2-Mint-flow_suSRo#_lu3C0) and [here](https://coda.io/d/Building-Keep_d-fmEgBNFVH/Sorin-2023-18-07-QA-tBTC-v2-dApp-issues_suMvR#_luV5k))

### Fixes:
---
#### 1. Change OutlineListItem color for dark mode - b274717d798801163ecb6f5df1a48c984cd191c5

Changes the color of the border for `MY ACTIVITY` items

<details>
  <summary>Screenshots</summary>

 Before:
 
<img width="302" alt="image" src="https://github.com/threshold-network/token-dashboard/assets/40306834/b02a14ef-cc14-413f-99f4-4207d1afd054">


After:
<img width="292" alt="image" src="https://github.com/threshold-network/token-dashboard/assets/40306834/0f65fece-bd74-4b2c-adc7-db106147f265">

</details>

---

#### 2. Fix darkmode color for BridgeProcessStep - b914db0f14a1ba8216490420bf62b0cb34313ff2

Changes the color of the title of BridgeProcessStep (for example `Unminting in progress`)

<details>
  <summary>Screenshots</summary>

 Before:
 
<TODO>


After:
<TODO>

</details>

---

#### 3. Fix color for TransactionDetailsItem (dark mode) - a573ff46cc4040bda6a138c49f2ce7b030eb8408

Makes the TransactionDetailsItem values lighter

<details>
  <summary>Screenshots</summary>

 Before:
 
<TODO>


After:
![image](https://github.com/threshold-network/token-dashboard/assets/40306834/d418d54f-1069-4d9e-b21b-6efa0beb2a09)

</details>

---

#### 4. Fix color for bridge process step subtitles  - 125b257cf3eca0a90b4f49d279fb841b34c79872

Makes the Bridge Process Step subtitle lighter.

<details>
  <summary>Screenshots</summary>

 Before:
![image](https://github.com/threshold-network/token-dashboard/assets/40306834/f8ae5add-1e4e-440d-9dff-f337bc31d01a)

<img width="255" alt="image" src="https://github.com/threshold-network/token-dashboard/assets/40306834/3dbcf700-bbf8-4804-a9b5-98ed15731920">



After:
![image](https://github.com/threshold-network/token-dashboard/assets/40306834/79eeb5ad-fb0c-407b-a3f9-ca6f891cd8b3)

![image](https://github.com/threshold-network/token-dashboard/assets/40306834/02664264-0298-4037-887d-39035764bda4)

</details>

---

#### 5. Fix the colors in Step component (dark mode) - 72da36ebb4d44e6862ec0b1cb500ea8baea52c11

<details>
  <summary>Screenshots</summary>

 Before:
<img width="251" alt="image" src="https://github.com/threshold-network/token-dashboard/assets/40306834/d09b9382-5ad8-4053-8ebd-e0f093647d29">

After:
![image](https://github.com/threshold-network/token-dashboard/assets/40306834/0ffae678-624b-4ffe-82ed-edc213919f39)

</details>

---

#### 6. Fix ProtocolHistory colors for dark mode - 7ed978529924a224a3336434989b34ead56c4943

<details>
  <summary>Screenshots</summary>

 Before:
<img width="506" alt="image" src="https://github.com/threshold-network/token-dashboard/assets/40306834/8484d723-bbe5-49b0-befa-77856819fffa">

After:
![image](https://github.com/threshold-network/token-dashboard/assets/40306834/5927e432-bf85-45b0-9b1e-08ddd050aab5)

</details>

---

#### 7. Fix dark mode issues for step 1 minting - 363f759f157a1b221222a56d737a1077dcdbed0f

- change body text to gray.300 for dark mode
- change placeholder text for the input to gray.500 for dark mode

<details>
  <summary>Screenshots</summary>

 Before:
<img width="497" alt="image" src="https://github.com/threshold-network/token-dashboard/assets/40306834/72545041-7d41-4a77-af8b-be105b8216c5">

After:
![image](https://github.com/threshold-network/token-dashboard/assets/40306834/6b804036-a5cf-42d8-8e30-2d12c957bdc3)

</details>

---

#### 8. Fix dark mode colors for tBTC Explorer - 9ae95cdd514eaffba63af5ec8fb758d03fc61f1f

<details>
  <summary>Screenshots</summary>

 Before:
<img width="780" alt="image" src="https://github.com/threshold-network/token-dashboard/assets/40306834/01d95992-d5a2-4683-81be-01b4376ddd92">

After:
<img width="770" alt="image" src="https://github.com/threshold-network/token-dashboard/assets/40306834/c71dcbf3-8754-4fda-89cc-7220696d81e6">

</details>

SorinQ commented:
Congrats!

Just two things:

1. The gradient's color here should be the same as the background which I think is `##333A47`

![Group 24](https://github.com/threshold-network/token-dashboard/assets/37292909/4a7197e3-cb41-4ecb-bc2e-9e7f325e6790)



2. A. Please use a different color for the stroke: the next light grey in the palette after the one we use for the rectangle background. If none is available, then just remove the stroke.

B. Please ensure that the 'tBTC AMOUNT' table title is the same color as the other titles. This comment also applies to the light version. Thanks

![Group 23](https://github.com/threshold-network/token-dashboard/assets/37292909/55d260c8-9fda-4fbb-ac63-1367732bdfc4)

r-czajkowski reviewed (commented):
Maybe let's use the `useColorModeValue` hook on the top of the component because in case of conditional rendering, we will get a warning in console. WDYT?

michalsmiarowski commented:
@SorinQ 

> 1. The gradient's color here should be the same as the background which I think is `#333A47`
> 
> ![Group 24](https://user-images.githubusercontent.com/37292909/262069782-4a7197e3-cb41-4ecb-bc2e-9e7f325e6790.png)

Fixed in eb03a430d411b9af8c4caff1b939a520404d8feb

> 2. A. Please use a different color for the stroke: the next light grey in the palette after the one we use for the rectangle background. If none is available, then just remove the stroke.

Fixed in f6fff3b9f948b16b2e43b2cce323dd2cfda07c99

Not sure if I did this one correctly. I've used `gray.500` for dark mode and keep `gray.100` for ligh mode. Let me know if that looks ok on the preview.


> B. Please ensure that the 'tBTC AMOUNT' table title is the same color as the other titles. This comment also applies to the light version. Thanks

Fixed in a986c200df0da04d8270c4cf8ed42403a8579a32

From the code it looks like the rest of the titles should be the same color as `tBTC AMOUNT` (instead of the other way around) so that's what I did. Let me know if that looks ok on the preview.

michalsmiarowski commented:
> Maybe let's use the `useColorModeValue` hook on the top of the component because in case of conditional rendering, we will get a warning in console. WDYT?

Good idea @r-czajkowski. Fixed it in 88bae6b7f00392bb2e0b79b2246cd963424e68b6
